.github/workflows/ci.yml

```
@@ -0,0 +1,57 @@
+name: CI
+
+on:
+  push:
+    branches: [devel,master]
+  pull_request:
+    branches: [devel,master]
```

```suggestion
  push:
      branches: [devel]
  pull_request:
```
I don't think we need to restrict running this to any branches for pull requests - we sometimes have pull request targeting release branches. For push that seems sensible, but we don't really use the master branch.